src/js/components/search.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react';
+import { render } from 'react-dom';
+import Autosuggest from 'react-autosuggest';
+import PropTypes from 'prop-types';
+import debounce from 'debounce';
+import isMobile from 'ismobilejs';
+
+let controller;
+const hasAbort = 'signal' in new Request('');
+const doFetch = hasAbort
+  ? (url, options = {}) => {
+    controller = new AbortController();
+    const { signal } = controller;
+    return fetch(url, { ...options, signal });
+  }
+  : fetch;
+const getSuggestions = (value, locale) => {
+  const inputValue = value.trim().toLowerCase();
+  const inputLength = inputValue.length;
+  if (!inputLength) {
+    return [];
+  }
+  return doFetch(`/search.php?q=${value}&l=${locale}`);
+};
+
+const getSuggestionValue = doc => doc.Page_Title;
+
+const renderSuggestion = doc => (
+  <a href={doc.Page_Link}>
+    <h6>{doc.Page_Title}</h6>
+  </a>
+);
+
+const shouldRenderSuggestions = value => value.trim().length > 2;
+
+const SearchInput = ({ locale, initialValue }) => {
```

Comment:
There's a logical flaw in this implementation: The "suggest" component actually shows result pages, rather than suggesting terms you might want to look for ("autocomplete"?). But when you select one of the suggestions, you aren't directed to that result, but it rather repeats the search with the title of that result page.

Reply:
All the auto complete results are wrapped in hrefs to the page they correspond to. When you say you "select" one of the suggestions, do you mean click on it?

Otherwise, if you're using keyboard and you hit enter on it, I do see that it puts the page title in the search. It should probably navigate to the page.

Reply:
Have changed to go the page link, rather than search, when an item is selected.